src/core/core/rpc_message.cpp

```diff
@@ -266,6 +290,9 @@ void message_ex::seal(bool fill_crc)
     dassert  (!_is_read && _rw_committed, "seal can only be applied to write mode messages");
     //dbg_dassert(header->body_length > 0, "message %s is empty!", header->rpc_name);
 
+    if (hdr_format != NET_HDR_DSN)
```

Comment:
You are right, the crc calculation and checking should now be done by individual message parsers, which will be implemented differently. Let's keep this in mind and fix it in the future, as you suggested.